FAQ.md

```diff
@@ -191,3 +191,12 @@ Possible mitigations are:
   kernel so as to use `vmalloc` instead of `kmalloc` for them.
 - Reduce memory pressure on the host.
 
+### Error appears when starting Firecracker.
+
+```
+Cannot create VMM.: Vm(VmFd(Error(12)))
```

Comment:
I think the error message has changed. 75457bfee948da0de6d4adc0c4376467d0abc614 has removed the punctuation sign and the underlying error type. I believe the error message is now:

```
Cannot create VMM: Error opening VM fd: VmFd(Os { code: 12, kind: Other, message: "Out of memory" })
```

I would not use the full error message in the FAQ. I'd rather say something in the lines of `If Firecracker exits logging an error with code X meaning Y, the root cause is Z`. In this case it would be code `12` meaning `Out of memory` with the root cause `not enough memory to run a microVM`.